### PR TITLE
[Document Intelligence] Generate inital version from Typespec with DPG

### PR DESCRIPTION
**Pull request by YalinLi0312**
This PR includes initial generated code from the latest tsp files. Samples and tests will be added seperately.

### CODE REVIEW

File: sdk/documentintelligence/azure-ai-documentintelligence/setup.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# --------------------------------------------------------------------------
+# coding: utf-8
+
+import os
+import re
+from setuptools import setup, find_packages
+
+
+PACKAGE_NAME = "azure-ai-documentintelligence"
+PACKAGE_PPRINT_NAME = "Azure Ai Documentintelligence"
```
Comment (catalinaperalta) [inline sdk/documentintelligence/azure-ai-documentintelligence/setup.py:16]: Nit: there's a config in tspconfig.yaml to set this value it's called `package-pprint-name`
```suggestion
PACKAGE_PPRINT_NAME = "Azure AI Document Intelligence"
```

File: sdk/documentintelligence/azure-ai-documentintelligence/tsp-location.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+directory: specification/cognitiveservices/DocumentIntelligence
+commit: 763e567abdb2154599a6d397444e60affc9cee17
+additionalDirectories: []
+repo: Azure/azure-rest-api-specs-pr
```
Comment (catalinaperalta) [inline sdk/documentintelligence/azure-ai-documentintelligence/tsp-location.yaml:4]: You should be able to update these values to the ones in the public repo if the tsp PR was already merged in

**Review by catalinaperalta — APPROVED** [review]
Approving since this is purely generated code. We need to fix the README before release, also in a few places in docs and setup.py the words Document Intelligence dont have a whitespace. Since you added ci.yml in this PR you can just run `/azp run prepare-pipelines` to create your pipelines now.

File: sdk/documentintelligence/azure-ai-documentintelligence/tsp-location.yaml (hunk 2 above)
Comment (YalinLi0312) [reply sdk/documentintelligence/azure-ai-documentintelligence/tsp-location.yaml:4]: Yup, the tsp PR is currently in open, will update once it merged.

File: sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    "DocumentSelectionMarkState",
+    "DocumentSignatureType",
+    "DocumentTableCellKind",
+    "FontStyle",
+    "FontWeight",
+    "LengthUnit",
+    "OperationKind",
+    "OperationStatus",
+    "ParagraphRole",
+    "SplitMode",
```
[the remark this answers is not in the document]
Comment (YalinLi0312) [reply sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/models/__init__.py:153]: It's a split mode enum used in `begin_classify_document`.

File: sdk/documentintelligence/azure-ai-documentintelligence/setup.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
```
Comment (johanste) [inline sdk/documentintelligence/azure-ai-documentintelligence/setup.py:49]: You are missing 3.12

**Review by johanste — APPROVED** [review]
Please update supported python versions.

**Pull request merged**